src/Pages/CheckOutPage.tsx

```diff
@@ -0,0 +1,5 @@
+const CheckOutPage = () => {
+	return <div>CheckOutPage</div>;
+};
+
+export default CheckOutPage;
```

Comment:
All newly created pages should have lower case file name.

Reply:
Sure, will change them